Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx

```diff
@@ -0,0 +1,268 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import { useState, Fragment, useEffect, useMemo } from 'react';
+import find from 'lodash/find';
+import formatMessage from 'format-message';
+import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
+import { ScrollablePane, ScrollbarVisibility } from 'office-ui-fabric-react/lib/ScrollablePane';
+import { Selection } from 'office-ui-fabric-react/lib/DetailsList';
+import {
+  DetailsList,
+  DetailsListLayoutMode,
+  SelectionMode,
+  CheckboxVisibility,
+  DetailsRow,
+} from 'office-ui-fabric-react/lib/DetailsList';
+import { BotTemplate } from '@bfc/shared';
+import { DialogWrapper, DialogTypes } from '@bfc/ui-shared';
+import { NeutralColors } from '@uifabric/fluent-theme';
+import { RouteComponentProps } from '@reach/router';
+import { IPivotItemProps, Pivot, PivotItem } from 'office-ui-fabric-react/lib/Pivot';
+import { csharpFeedKey, nodeFeedKey } from '@botframework-composer/types';
+
+import { DialogCreationCopy, EmptyBotTemplateId, feedDictionary, QnABotTemplateId } from '../../../constants';
+import httpClient from '../../../utils/httpUtil';
+
+import { TemplateDetailView } from './TemplateDetailView';
+
+// -------------------- Styles -------------------- //
+
```

Comment:
`detailsListContainer`, `templateDetailContainer`, `pickerContainer` don't appear to be responsive. Is this by design?

Reply:
Its parent up the div chain allows for decent responsiveness at current state with these existing rules. That being said there is a task item for design iterations on this modal (responsiveness details, boilerplate readme structure, general pixel corrections). I am going to leave this as is and revisit when we do the smaller PR design iterations.

Reply:
We can use the <Stack> element, it does a nice job in these cases.